src/common/types/value.cpp

```diff
@@ -502,6 +517,9 @@ void Value::serialize(FileInfo* fileInfo, uint64_t& offset) const {
     case PhysicalTypeID::INT16: {
         SerDeser::serializeValue(val.int16Val, fileInfo, offset);
     } break;
+    case PhysicalTypeID::INT8: {
+        SerDeser::serializeValue(val.int8Val, fileInfo, offset);
```

Comment:
Keep switch cases in this file, they are used for literalExpression in UDF. You don't need to add a test for this.